src/DGtal/helpers/Shortcuts.h

```diff
@@ -0,0 +1,2846 @@
+	  if ( ! c2i.count( primal_vtx ) ) {
+	    result.push_back( primal_vtx );
+	    c2i[ primal_vtx ] = n++;
+	  }
+	}
+      }
+      return result;
+    }
+
+    /// Given any digital surface, returns the vector of its pointels.
+    ///
+    /// @note The order of pointels is given by the default traversal
+    /// of the surfels of the surface, where the 4 pointels of each
+    /// surfel are visited in order.
+    ///
+    /// @note If you wish to consider the primal digital surface, and
+    /// visits pointels as vertices of this graph in
+    /// breadth-first/depth-first order, the best is to build first a
+    /// PolygonalSurface and then use specialized visitors.
+    ///
+    /// @tparam TDigitalSurfaceContainer either kind of DigitalSurfaceContainer
+    ///
+    /// @param[in] surface a smart pointer on a (light or not) digital surface (e.g. DigitalSurface or LightDigitalSurface).
+    ///
+    /// @return a range of pointels as a vector.
+    template <typename TDigitalSurfaceContainer>
+    static PointelRange
+    getPointelRange
+    ( CountedPtr< ::DGtal::DigitalSurface<TDigitalSurfaceContainer> > surface )
+    {
+      Cell2Index c2i;
+      return getPointelRange( c2i, surface );
+    }
+
+    /// Given any surfel, returns its 4 pointels in ccw order.
+    ///
+    /// @see getPrimalVertices
+    ///
+    /// @param[in] K the Khalimsky space
+    /// @param[in] surfel any surfel that lives in the Khalimsky space
+    ///
+    /// @return a range of pointels as a vector.
+    static PointelRange
+    getPointelRange
+    ( const KSpace& K, const SCell& surfel )
+    {
+      return getPrimalVertices( K, surfel, true );
+    }
+    
+    /// Given any digital surface, returns a vector of surfels in
+    /// some specified order.
+    ///
+    /// @tparam TDigitalSurfaceContainer either kind of DigitalSurfaceContainer
+    ///
+    /// @param[in] surface a smart pointer on a (light or not) digital surface (e.g. DigitalSurface or LightDigitalSurface).
+    ///
+    /// @param[in] params the parameters:
+    ///   - surfaceTraversal  ["Default"]: "Default"|"DepthFirst"|"BreadthFirst": "Default" default surface traversal, "DepthFirst": depth-first surface traversal, "BreadthFirst": breadth-first surface traversal.
+    ///
+    /// @return a range of surfels as a vector.
+    template <typename TDigitalSurfaceContainer>
+    static SurfelRange
+    getSurfelRange
+    ( CountedPtr< ::DGtal::DigitalSurface<TDigitalSurfaceContainer> > surface,
+      const Parameters&   params = parametersDigitalSurface() )
+    {
+      return getSurfelRange( surface, *( surface->begin() ), params );
+    }
+
+    /// Given a light digital surface, returns a vector of surfels in
+    /// some specified order.
+    ///
+    /// @tparam TDigitalSurfaceContainer either kind of DigitalSurfaceContainer
+    ///
+    /// @param[in] surface a smart pointer on a (light or not) digital surface (e.g. DigitalSurface or LightDigita
```

Comment:
```suggestion
	  surfels.push_back( surfelmap[ idx ] );
```

Reply:
Is it your editor that create this weird indentation ?

Reply:
it is a bug in GitHub's suggestion. I just wanted to add two spaces before `surfel` (there is a for loop above)